### PR TITLE
Changed storage diagram and new issues for delete function

### PR DESCRIPTION
**Pull request by NoorSarrah**

### CODE REVIEW

**Review by Chiamjiaen — APPROVED** [review]
looks good, ready for merging.

**Review by blank-bank — COMMENTED** [review]
Looks good.

File: src/main/java/seedu/duke/Constants.java [hunk 1]
```diff
@@ -115,7 +115,7 @@
     public static final String PATIENT_DELETE_FORMAT = "delete/[Patient ID]";
     public static final String APPOINTMENTS_DELETE_FORMAT = "delete/[DoctorID/Appointment ID]";
     public static final String SCHEDULES_DELETE_FORMAT = "delete/[Nurse ID]/[Date (DDMMYYYY)]";
-    public static final String INVENTORY_DELETE_FORMAT = "delete/[Drug Name]";
+    public static final String INVENTORY_DELETE_FORMAT = "delete/[Drug Name]/[Quantity]";
```
Comment (AlexanderTanJunAn) [inline src/main/java/seedu/duke/Constants.java:118]: Looks good to me. Makes error message clearer.

**Review by AlexanderTanJunAn — COMMENTED** [review]
LGTM

**Review by MingShun98 — COMMENTED** [review]
Looks good to me, remember to check the text style for Java CI

**Pull request merged**